Sources/SwiftDiagnostics/DiagnosticsFormatter.swift

```diff
@@ -139,10 +139,11 @@ public struct DiagnosticsFormatter {
     switch message.severity {
     case .error:
       let annotation = ANSIAnnotation(color: .red, trait: .bold)
-      return annotation.applied(to: message.message)
+      return annotation.applied(to: "error: \(message.message)")
     case .warning:
-      let annotation = ANSIAnnotation(color: .yellow)
-      return annotation.applied(to: message.message)
+      let color = ANSIAnnotation(color: .yellow)
+      let prefix = color.withTrait(.bold).applied(to: "warning: ")
```

Comment:
Just to be clear: It’s intentional that errors have both the prefix and the the message in bold but warnings only have a bold prefix? I’m fine with either way but just want to make sure the current design is a conscious decision.